README.md

```diff
@@ -60,6 +60,7 @@ make build
 * [x] Array(FixedString)
 * [x] Array(DateTime(UInt32), Date(UInt16))
 * [x] Nullable
+* [x] [ElasticDate](https://www.elastic.co/guide/en/elasticsearch/reference/current/date.html) => Int64 (2019-12-16T12:10:30Z => 1576498230)
```

Comment:
I agree to rename `ElasticDate` => `ElasticDateTime` 
it seems right

but I believe we should use int64 but not UInt32 
because for example:
if someone will handle historical events or predict future events they will be limited by the time slot
```
TZ=UTC date -d @0          
Thu 01 Jan 1970 12:00:00 AM UTC
TZ=UTC date -d @4294967295 
Sun 07 Feb 2106 06:28:15 AM UTC
```

besides the function `Unix` is returns int64
```
func (t Time) Unix() int64 {
	return t.unixSec()
}
```

Reply:
Agree that int64 will be better. (I don't know ClickHouse maintainer will how to solve it when 2106 happens, It's just an alias which DateTime is UInt32 )